kbase-extension/static/kbase/js/common/parameterSpec.js

```diff
@@ -237,13 +234,10 @@ define([], () => {
             if (value === undefined || value === null) {
                 return true;
             }
-            if (
```

Comment:
sonar cloud optimisation

Reply:
The `!!` seems a little redundant. Shouldn't that whole expression resolve to a boolean? I guess it makes it a little easier to see at a glance.